libs/component-library/src/lib/svg/svg.tsx

```diff
@@ -1,57 +1,23 @@
 import React from 'react';
 import {
   ColorProps,
-  composeRestyleFunctions,
+  createRestyleComponent,
   spacing,
   SpacingProps,
-  useRestyle,
   useTheme,
 } from '@shopify/restyle';
 import Reanimated from 'react-native-reanimated';
 import Svg, { SvgProps } from 'react-native-svg';
 import { FxTheme } from '../theme/theme';
 
-const ReanimatedSvg = Reanimated.createAnimatedComponent(Svg);
-
 export type FxSvgProps = ColorProps<FxTheme> &
   SpacingProps<FxTheme> &
   Omit<SvgProps, 'color'>;
 
-const restyleFunctions = composeRestyleFunctions<
-  FxTheme,
-  Omit<
-    FxSvgProps,
-    | 'color'
-    | 'fill'
-    | 'opacity'
-    | 'fontSize'
-    | 'fontStyle'
-    | 'fontWeight'
-    | 'letterSpacing'
-  >
->([spacing]);
+const RestyledSvg = createRestyleComponent<FxSvgProps, FxTheme>([spacing], Svg);
```

Comment:
Shouldn't this include `[color, spacing]`?

Reply:
In this case I think it's not needed as it's coming from the `useTheme` hook instead and being applied to the SVG using the fill prop. I could be wrong.

Reply:
In this case I think it's not needed as it's coming from the useTheme hook instead and being applied to the SVG using the fill prop. I could be wrong.

Reply:
Yep, color is getting converted with theme values from `useTheme`